doc/user/user/search/index.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: default
+title: "Search"
+search_title: "IRIDA Search Overview"
+description: "An overview of the features present on the IRIDA global search."
+---
+
+IRIDA Global Search Overview
+============================
+{:.no_toc}
+
+The IRIDA global search is design to quickly find projects and samples that you have access to anywhere in IRIDA.
```

Comment:
Chang `design` to `designed`

Reply:
Updated in 6db0f24b8137f7fbfc0be48cbb6dd58bd3e55be0